src/routes/CurrencyDescription.jsx

```diff
@@ -0,0 +1,93 @@
+import { NavLink, useParams } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { BsFillArrowLeftCircleFill } from 'react-icons/bs';
+import { useState } from 'react';
+import { changeMoney } from '../redux/money/moneySlice';
+
+const CurrencyDescription = () => {
+  const { id } = useParams();
+  const currencies = useSelector((state) => state.currency.allCurrencies);
+  const price = useSelector((state) => state.currency.usdPrice);
+  const money = useSelector((state) => state.money.value);
+  const dispatch = useDispatch();
+  const [extraCurrencies, setExtraCurrencies] = useState([]);
+  const otherCurrencies = async () => {
+    const response = await fetch(`https://cdn.jsdelivr.net/gh/fawazahmed0/currency-api@1/latest/currencies/${id}.json`);
+    const data = await response.json();
+    setExtraCurrencies(data);
+    return data;
+  };
+  otherCurrencies();
+  const handleMoney = (e) => {
+    dispatch(changeMoney(e.target.value));
+  };
+
+  let color = 0;
+  return (
+    <div>
+      <div className="currency-info">
+        <h1>{id}</h1>
+        <p>
+          Name:
+          {' '}
+          {currencies[id]}
+        </p>
+        <input className="money" type="number" onChange={handleMoney} placeholder="How much money? (USD)" />
+        <p>
+          With
+          {' '}
+          {money}
+          {' '}
+          USD dollar, you can buy:
+          {' '}
+        </p>
+        <p>
+          {price[id] * money}
+          {' '}
+          {currencies[id]}
+          s
+        </p>
+        <NavLink className="back-link" to="/">
+          <i className="back-arrow">
+            <BsFillArrowLeftCircleFill />
+          </i>
+          {' '}
+          Back
+        </NavLink>
+      </div>
+      <hr className="divisor" />
```

Comment:
- [ ] As required for the project to correctly match the provided design, I think would be better if you move the `Back` **Arrow** button at his expected place like on the design.

| Template Design | Your Design |
| ---------------------- | -------------------- |
| ![Screenshot from 2023-02-10 21-59-28](https://user-images.githubusercontent.com/83042742/233220207-8cb5563c-9a75-4181-bf52-9b31c66ebefe.png) | ![Screenshot from 2023-04-20 01-14-44](https://user-images.githubusercontent.com/83042742/233220050-5d810969-cca3-4063-a36f-4aa2fc7dffe1.png) |